alchemy/src/cloudflare/bound.ts

```diff
@@ -61,8 +62,10 @@ export type Bound<T extends Binding> = T extends _DurableObjectNamespace<
                                   ? Fetcher
                                   : T extends _Ai<infer M>
                                     ? Ai<M>
-                                    : T extends Self
-                                      ? Service
-                                      : T extends Json<infer T>
-                                        ? T
-                                        : Service;
+                                    : T extends _Images
+                                      ? Fetcher
```

Comment:
I don't think Fetcher is right, see https://developers.cloudflare.com/images/transform-images/bindings/

example:
```ts
// Fetch the watermark from Workers Assets, R2, KV etc
const watermark: ReadableStream = ...

// Fetch the main image
const image: ReadableStream = ...

const response = (
  await env.IMAGES.input(image)
    .draw(
        env.IMAGES.input(watermark)
          .transform({ width: 32, height: 32}),
        { bottom: 32, right: 32 }
    )
    .output({ format: "image/avif" })
).response()

return response;
```